pyradiosky/skymodel.py

```diff
@@ -890,14 +929,14 @@ def skymodel_to_array(sky):
         )
         fieldnames.extend(["frequency", "reference_frequency"])
         fieldtypes.extend(["f8"] * 2)
-        fieldshapes.extend([()] * 3)
+        fieldshapes.extend([(n_stokes,)] + [()] * 3)
```

Comment:
I think this should be `fieldshapes.extend([(n_stokes,)] + [()] * 2)` -- The third one was for the flux column, which is now taken care of by the one that's `n_stokes` wide.

Reply:
The change is made.